src/pages/NoticePage.tsx

```diff
@@ -0,0 +1,34 @@
+import { styled } from 'styled-components';
+
+import Article from '../components/Article';
+import Header from '../components/Notice/Header';
+import NoticeItem from '../components/Notice/NoticeItem';
+
+import useFetchNotices from '../hooks/useFetchNotice';
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const NoticeList = styled.div`
+`;
+
+export default function NoticePage() {
+  const notices = useFetchNotices();
+
+  return (
+    <Wrapper>
+      <Header shadow="true">알림 센터</Header>
```

Comment:
shadow props가 boolean 타입으로 관리되면 이런식으로 사용할 수 있어요~
```suggestion
      <Header shadow>알림 센터</Header>
```